mupdf-sys/wrapper.c

```diff
@@ -1117,6 +1117,37 @@ fz_pixmap *mupdf_display_list_to_pixmap(fz_context *ctx, fz_display_list *list,
     return pixmap;
 }
 
+fz_buffer *mupdf_display_list_to_svg(fz_context *ctx, fz_display_list *list, fz_matrix ctm, fz_cookie *cookie, mupdf_error_t **errptr)
+{
+    fz_rect mediabox = fz_bound_display_list(ctx, list);
+    fz_device *dev = NULL;
+    fz_buffer *buf = NULL;
+    fz_output *out = NULL;
+    fz_var(out);
+    fz_var(dev);
+    fz_var(buf);
+    fz_rect tbounds = mediabox;
+    tbounds = fz_transform_rect(tbounds, ctm);
+    fz_try(ctx)
+    {
+        buf = fz_new_buffer(ctx, 1024);
+        out = fz_new_output_with_buffer(ctx, buf);
+        dev = fz_new_svg_device(ctx, out, tbounds.x1 - tbounds.x0, tbounds.y1 - tbounds.y0, FZ_SVG_TEXT_AS_PATH, 1);
+        fz_run_display_list(ctx, list, dev, ctm, tbounds, cookie);
```

Comment:
I'm not completely sure if mediabox or tbounds should be used for the area here.

Reply:
Not sure either, I guess we'll know better when it gets used in real world applications.